src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const fixRobot = {};
+
+  const arrValues = Object.values(robot);
+  const arrKeys = Object.keys(robot);
+
+  for (const key in robot) {
+    if (arrValues.filter(n => n === robot[key]).length > 1
+    || arrKeys.filter(n => n === key).length > 1) {
```

Comment:
You only need condition for values.
You also can use [hasOwnProperty](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty) method instead of Object.values and filter

Reply:
Fixed. Did you mean it?
And still another option. Check after reverse:

  if (Object.keys(fixRobot).length
    !== Object.keys(robot).length) {
    return null;
  }